repokid/cli/reactor_cli.py

```diff
@@ -0,0 +1,192 @@
+
+    if not role_id:
+        return (False, 'Unable to find role {} in account {}'.format(message.role_name, message.account))
+    else:
+        role_data = dynamo.get_role_data(dynamo_table, role_id, fields=['RepoableServices'])
+        repoable_services = role_data['RepoableServices']
+        return (True, 'Repoable services from role {} in account {}: {}'.format(message.role_name, message.account,
+                                                                                repoable_services))
+
+
+def list_role_rollbacks(dynamo_table, message, config=None):
+    role_id = dynamo.find_role_in_cache(dynamo_table, message.account, message.role_name)
+
+    if not role_id:
+        return (False, 'Unable to find role {} in account {}'.format(message.role_name, message.account))
+    else:
+        role_data = dynamo.get_role_data(dynamo_table, role_id, fields=['Policies'])
+        return_val = 'Restorable versions for role {} in account {}\n'.format(message.role_name, message.account)
+        for index, policy_version in enumerate(role_data['Policies']):
+            return_val += '({:>3}):  {:<5}     {:<15}  {}\n'.format(index, len(str(policy_version['Policy'])),
+                                                                    policy_version['Discovered'],
+                                                                    policy_version['Source'])
+        return (True, return_val)
+
+
+def opt_out(dynamo_table, message, config=None):
+    if not message.reason or not message.requestor:
+        return (False, 'Reason and requestor must be specified')
+
+    role_id = dynamo.find_role_in_cache(dynamo_table, message.account, message.role_name)
+
+    if not role_id:
+        return (False, 'Unable to find role {} in account {}'.format(message.role_name, message.account))
+
+    role_data = dynamo.get_role_data(dynamo_table, role_id, fields=['OptOut'])
+    if 'OptOut' in role_data and role_data['OptOut']:
+
+        return (False, 'Role {} in account {} is already opted out by {} for reason {} until {}'.format(
+            message.role_name, message.account, role_data['OptOut']['owner'], role_data['OptOut']['reason'],
+            time.strftime('%m/%d/%y', time.localtime(role_data['OptOut']['expire']))))
+
+    else:
+        current_dt = datetime.datetime.fromtimestamp(time.time())
+        expire_dt = current_dt + datetime.timedelta(config['opt_out_period_days'])
+        expire_epoch = int((expire_dt - datetime.datetime(1970, 1, 1)).total_seconds())
+        new_opt_out = {'owner': message.requestor, 'reason': message.reason, 'expire': expire_epoch}
+        dynamo.set_role_data(dynamo_table, role_id, {'OptOut': new_opt_out})
+        return (True, 'Role {} in account {} opted-out until {}'.format(message.role_name, message.account,
+                                                                        expire_dt.strftime('%m/%d/%y')))
+
+
+def remove_opt_out(dynamo_table, message, config=None):
```

Comment:
Seems weird to me that you're importing CONFIG from repokid, but then also passing it to every method.